### PR TITLE
add basis module

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: examples/basis/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* global fetch, document, URL */
+import {load} from '@loaders.gl/core';
+
+
+const elStatus = document.getElementById('status');
+const elProgress = document.getElementById('progress');
+const elImage = document.getElementById('img');
+
+/*
```
Comment (ibgreen) [inline examples/basis/app.js]: Add TODO or uncomment?

File: examples/basis/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<script type="module">
+  import './app.js';
+</script>
+<body>
+  <main>
+    <!-- Forked from github AnthumChris/fetch-progress-indicators under MIT license -->
```
Comment (ibgreen) [inline examples/basis/index.html]: Remove irrelevant attribution...

File: modules/basis/src/basis-loader.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const EXTENSIONS = ['basis'];
+
+export default {
+  name: 'basis',
+  extensions: EXTENSIONS,
+  parse: x => console.log('parse', x),
+  loadAndParse: x => console.log('loadAndParse', x)
```
Comment (ibgreen) [inline modules/basis/src/basis-loader.js]: You don't need loadAndParse. Just `parse` and maybe `parseSync`

**Comment by ibgreen** [comment]
Also add placeholders to docs/tests/benchmarks

File: examples/basis/app.js (hunk 1 above)
Comment (georgios-uber) [reply examples/basis/app.js]: Updated it

File: examples/basis/index.html (hunk 2 above)
Comment (georgios-uber) [reply examples/basis/index.html]: done

File: modules/basis/src/basis-loader.js (hunk 3 above)
Comment (georgios-uber) [reply modules/basis/src/basis-loader.js]: ok

**Pull request merged**